how-to/how-to-use-rootless-kata-containers-with-podman.md

```diff
@@ -0,0 +1,170 @@
+  - If using CentOS 7 `newuidmap` and `newgidmap` do not exist and can be installed by:
+    ``` bash
+    $ (git clone https://github.com/shadow-maint/shadow; cd shadow; ./autogen.sh --prefix=/usr --enable-man; make && sudo make -C src install)
+    $ sudo bash -c 'echo 10000 > /proc/sys/user/max_user_namespaces'
+    $ sudo bash -c "echo $(whoami):110000:65536 > /etc/subuid"
+    $ sudo bash -c "echo $(whoami):110000:65536 > /etc/subgid"
+    ```
+- Golang 1.12
+
+## Installation
+
+Refer to the following table for the __minimum__ required software versions and the installation instructions:
+
+| Component       | Version | Install Instructions|
+| ----------------|:-------:|---------------------|
+| Podman          | WIP     | https://github.com/containers/libpod/blob/master/install.md
+| slirp4netns     | 0.4.0   | https://github.com/rootless-containers/slirp4netns#quick-start
+| Kata Containers | WIP     | https://github.com/kata-containers/documentation/blob/master/install/README.md
+
+> **NOTE:**
+> If installing Podman with a package manager, there is usually no need to install slirp4netns separately.
+
+## Configuration
+Now that Kata Containers and Podman have been installed, they need to be configured for rootless execution.
+
+#### 1. Disable SELinux
+If SELinux is installed and enabled, it needs to be disabled with the following command (Kata Containers [does not support SELinux](https://github.com/kata-containers/documentation/blob/master/Limitations.md#selinux-support)
+
+> **Warning:**
+> The following command may differ depending on the distro being used
+
+```bash
+$ [ -f /etc/selinux/config ] && sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config
+```
+
+#### 2. Add user to KVM group
+If running a KVM based hypervisor, the user running the workload needs to be added to the KVM group
+```bash
+$ sudo usermod -a -G kvm $USER
+```
+
+#### 3. Reboot
+Reboot the system for the changes to take effect (a reboot is required when disabling SELinux, while logging out and back in is enough to have that user joining the `KVM` group).
+
+You can now verify if the configuration is correct:
+
+* (if installed) SELinux should have been disabled:
+```bash
+$ getenforce
+Disabled
+```
+
+* The user should be in the `kvm` group:
+```
+$ groups | grep -ow kvm
+kvm
+```
+
+#### 4. Setup Kata configuration files
+If the Kata `configuration.toml` file does not exist in `/etc` do the following:
+```bash
+$ sudo install -D -o ${USER} -g root -m 0640 /usr/share/defaults/kata-containers/configuration.toml /etc/kata-containers
+```
+
+Or, if the file exists, but is not readable by the user:
+```bash
+$ sudo chown -R a+r /etc/kata-containers/
+```
+
+#### 5. Disable `vhost-net`
+Disable `vhost-net` in the Kata configuration file, by commenting out the `disable_vhost_net` line:
```

Comment:
I like the idea of automatically disabling vhost-net when running rootless, since there is a way of checking whether execution is rootless or not. Not sure if there would be any other issues with doing that! We could open an issue for discussion on this!